.github/workflows/kind-e2e.yaml

```diff
@@ -120,10 +143,15 @@ jobs:
 
       - name: Run e2e Tests
         run: |
+          source test/e2e-common.sh
+
           export KO_FLAGS="--platform=linux/amd64"
           export SYSTEM_NAMESPACE=knative-eventing
           export CLUSTER_DOMAIN=${CLUSTER_SUFFIX}
 
+          # Setup auth config for KafkaChannel. Uses CHANNEL_AUTH_SCENARIO env var.
+          setup_kafka_channel_auth
+
```

Comment:
Can we make this an `hack/run.sh` target and use that instead here?

So that we can use that target for development too.

Reply:
Would you mind if I do that separately?

Reply:
sounds good!

/lgtm